modules/table/src/main/java/org/apache/ignite/internal/table/distributed/disaster/ManualGroupUpdateRequest.java

```diff
@@ -79,31 +120,164 @@ public CompletableFuture<Void> handle(DisasterRecoveryManager disasterRecoveryMa
         HybridTimestamp msSafeTime = disasterRecoveryManager.metaStorageManager.timestampByRevision(msRevision);
 
         int catalogVersion = disasterRecoveryManager.catalogManager.activeCatalogVersion(msSafeTime.longValue());
+
+        if (this.catalogVersion != catalogVersion) {
+            return CompletableFuture.failedFuture(new IgniteException("Foo"));
```

Comment:
Why "Foo" not "Bar" or "Duck" ? =)

Reply:
That's the default for me, "bar" goes next. And I only swear when I see a bad code :)